openedx/openedx-platform
# feat: implement tag object rest api in cms

## Description

This PR implements the tagging REST API for creating/updating object tags in Courses and XBlocks.

## Supporting Information

* Closes https://github.com/openedx/modular-learning/issues/98

## Testing instructions
* Ensure that the tests cover the expected behavior of the view as described in the related issue.
___
Private ref: [FAL-3499](https://tasks.opencraft.com/browse/FAL-3499)
